ui/components/multichain/ramps-card/ramps-card.js

```diff
@@ -85,13 +89,14 @@ export const RampsCard = ({ variant }) => {
   }, [currentLocale, chainId, nickname, trackEvent]);
 
   const onClick = useCallback(() => {
-    openBuyCryptoInPdapp();
+    openBuyCryptoInPdapp(chainId);
```

Comment:
Do we need to update openBuyCryptoInPdapp as well?

Reply:
Also a leftover from my initial PR for bitcoin account overview, let me remove it for now.